File: Backend/UserDataService/src/main/java/com/user/dto/UserAfterLogin.java
package com.user.dto;

import java.time.LocalDate;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@NoArgsConstructor
@AllArgsConstructor
@Data
public class UserAfterLogin {
	private String name;
	private String email;
	private String city;
	private LocalDate creationDate;
	private Double totalFootprint;
	private Double sixMonthRewardPoints;
	private Double totalRewardPoints;
}

File: FrontEnd/Self-Tracker/src/app/models/user-after-login.ts
export class UserAfterLogin {
    name: string;
    email: string;
    city: string;
    creationDate: Date;
    totalFootprint: number;
    sixMonthRewardPoints: number;
    totalRewardPoints: number;
  
    constructor(
      name: string,
      email: string,
      city: string,
      creationDate: Date,
      totalFootprint: number,
      sixMonthRewardPoints: number,
      totalRewardPoints: number
    ) {
      this.name = name;
      this.email = email;
      this.city = city;
      this.creationDate = creationDate;
      this.totalFootprint = totalFootprint;
      this.sixMonthRewardPoints = sixMonthRewardPoints;
      this.totalRewardPoints = totalRewardPoints;
    }
  }